library/src/engine.cxx

```diff
@@ -52,32 +52,43 @@ engine::engine(
   // Ensure all lib initialization is done (once)
   detail::init::initialize();
 
+  fs::path cachePath;
 #if defined(_WIN32)
-  static constexpr const char* CACHE_ENV_VAR = "LOCALAPPDATA";
+  const char* appData = std::getenv("APPDATA");
```

Comment:
Why is it `APPDATA` instead of `LOCALAPPDATA` now?

Reply:
mistake on my part, it should be LOCALAPPDATA afaiu